packages/i18n/src/resources/en.json

```
@@ -174,5 +176,11 @@
   "Cloud Workspace Description": "All data will be synchronized and saved to the AFFiNE account <1>{{email}}</1>",
   "Download data Description1": "It takes up more space on your device.",
   "Download data Description2": "It takes up little space on your device.",
-  "Enabled success": "Enabled success"
+  "Enabled success": "Enabled success",
+  "Export AFFINE backup file (coming soon)": "Export AFFINE backup file (coming soon)",
```

I think the "(coming soon)" of the key can be removed
```suggestion
  "Export AFFINE backup file": "Export AFFINE backup file (coming soon)",
```